src/display/text_layer.js

```diff
@@ -679,8 +666,11 @@ const renderTextLayer = (function renderTextLayerClosure() {
         }
         textDiv.style.transform = transform;
       }
-      this._textDivProperties.set(textDiv, textDivProperties);
+
       this._container.appendChild(textDiv);
+      if (textDivProperties.hasEOL) {
+        textDiv.appendChild(this._document.createElement("br"));
```

Comment:
Why not just `textDiv.appendChild(document.createElement("br"))` instead, since this is only creating a *simple* element (which isn't accessed further)?